src/front/app/components/ChallengeDetail/index.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import ChallengeDetail from './ChallengeDetail';
+
+export default () => (
+    <div>
+        <ChallengeDetail />
```

Comment:
container for one component? Maybe it§s better to move ChallengeDetail component into this file?